internal/pkg/installer/installer.go

```diff
@@ -70,16 +54,6 @@ func (i *Installer) Install() (err error) {
 		return nil
 	}
 
-	// Verify that the target device(s) can satisify the requested options.
-
-	if err = VerifyBootDevice(i.data); err != nil {
```

Comment:
These need to be moved into the `NewManifest` function so that we can verify the machine config early and avoid nil pointers.